modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java

```diff
@@ -713,9 +713,8 @@ private Sku _updateSKU(CPInstance cpInstance, Sku sku) throws Exception {
 			serviceContext.setExpandoBridgeAttributes(expandoBridgeAttributes);
 		}
 
-		long replacementCProductId = cpInstance.getReplacementCProductId();
-		String replacementCPInstanceUuid =
-			cpInstance.getReplacementCPInstanceUuid();
+		long replacementCProductId = 0;
+		String replacementCPInstanceUuid = StringPool.BLANK;
```

Comment:
What if they do not pass replacement info at all? It should fall back to the one already stored in the DB